DayPicker.js

```diff
@@ -9,7 +9,7 @@ var Weekday = require('./lib/src/Weekday').default;
 var Navbar = require('./lib/src/Navbar').default;
 var PropTypes = require('./lib/src/PropTypes');
 
-module.exports = DayPicker.default || DayPicker;
+module.exports.DayPicker = DayPicker.default || DayPicker;
```

Comment:
If I go back to `module.exports = DayPicker.default || DayPicker`, my sample Typescript application doesn't load the DayPicker anymore.

I get this React error message:
`
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of...
`

I reverted to your code and I tried the following to solve the React error from above:

- Set the keyword `default` in `export default class DayPicker ...` in index.d.ts
- Tried `import { default as DayPicker}` from "react-day-picker";` in main.tsx of sample application 

Both of them did not work and I get the same error.

From my understanding, `module.exports = DayPicker.default || DayPicker` is like assigning the default value to object `exports`.

Would you have an idea on how to import this so it works in Typescript?